brainstormforce/ultimate-addons-for-gutenberg
# Fix: Table of Contents - TOC not scrolling to heading when smooth scroll is disabled.

### Description


Trello - https://trello.com/c/jMknl58P/274-fix-the-table-of-content-not-working-when-the-smooth-scroll-is-disabled

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop TOC and disabled the smooth scroll heading option.
- TOC not scrolling to a specific heading.
- Switch to `toc-not-scrolling-when-smooth-scroll-disabled` and check again if it's working or not

### Checklist:
